scripts/chronic_disease_indicators_in_us__cdi_.json

```diff
@@ -0,0 +1,187 @@
+{
+    
+    "citation": "",
+    "encoding": "utf-8",
+    "homepage": "https://data.world/cdc/us-chronic-disease-indicators",
+    "keywords": [
+        "chronic",
+        "diabetes",
+        "hypertension",
+        "cardiovascular disease",
+        "public health"
+    ],
+    "name": "chronic-disease-indicators-in-us",
```

Comment:
rename to `chronic-disease-indicators-us` and file name to chronic_disease_indicators_us.json